torch/autograd/profiler.py

```diff
@@ -112,8 +112,8 @@ class profile:
     Args:
         enabled (bool, optional): Setting this to False makes this context manager a no-op.
 
-        use_cuda (bool, optional): Enables timing of CUDA events as well using the cudaEvent API.
-            Adds approximately 4us of overhead to each tensor operation.
+        use_device (str, optional): Enables timing of device events.
```

Comment:
Could we also comment the list of valid options so far? Such as cuda, xpu, privateuseone.

Reply:
yes we can also check for it being "in VAID_DEVICE_OPTIONS"

Reply:
Thank you for comments.